conf/messages

```diff
@@ -811,9 +811,11 @@ gt.home=Go back to home
 gt.vote.ok=Your vote has been recorded.
 gt.youcan=You can now:
 gt.nomore=There is no more talk to review, congratulation you did it!
-gt.nextTalk=Continue and evaluate next talk
+gt.nextTalkTrackAndType=Continue and evaluate another {0} for track {1}
```

Comment:
Not sure why in English we don't highlight the parameters with []
It's perhaps more readable

```suggestion
gt.nextTalkTrackAndType=Continue and evaluate another [{0}] for track [{1}]
```

Reply:
Yep wondered too, I didn't wanted to change this just in case this was willed ... 
=> will change this in a single commit

Reply:
Just stumbled upon the `showVotesForProposal.scala.html` called through the `GoldenTicketController`

Currently, this page is going to fail because of the `gt.nextTalk` message entry removal

Do you think it is ok to update this template (and the corresponding controller) with the same changes I did on the `CFPAdmin` controller ?
Or you would like to prevent changes from the GT side as much as possible ?

Reply:
I would say that we should do the same for our GoldenTickets too
But I am not sure how to test it cc @nicmarti

Reply:
Fixed commits on the GT side with following fixup commits
f2f8449d8cd3c0258a58eb1cda98dfc91aebf043
78b931f37e57f6e492f351eeb0960d4a4988df19
686045f590bfcc3ecf875884c96a5db303c8083f
a9c3e5cc40a31a7b6fc551ff9706f14398cde1e4

Reply:
(sorry marked as resolved while there is still a pending question for @nicmarti :-) )
=> unresolved it